src/components/flow-item/flow-item.stories.ts

```diff
@@ -0,0 +1,219 @@
+import { boolean, select, text } from "@storybook/addon-knobs";
+import {
+  Attributes,
+  createComponentHTML as create,
+  Attribute,
+  filterComponentAttributes,
+  themesDarkDefault
+} from "../../../.storybook/utils";
+import { ATTRIBUTES } from "../../../.storybook/resources";
+import readme from "./readme.md";
+import { SLOTS, TEXT } from "./resources";
+import { html } from "../../../support/formatting";
+import { storyFilters } from "../../../.storybook/helpers";
+
+export default {
+  title: "Components/Flow Item",
+  parameters: {
+    notes: readme
+  },
+  ...storyFilters()
+};
+
+const createAttributes: (options?: { exceptions: string[] }) => Attributes = ({ exceptions } = { exceptions: [] }) => {
+  const { scale } = ATTRIBUTES;
+
+  return filterComponentAttributes(
+    [
+      {
+        name: "dismissed",
+        commit(): Attribute {
+          this.value = boolean("dismissed", false);
+          delete this.build;
+          return this;
+        }
+      },
+      {
+        name: "disabled",
+        commit(): Attribute {
+          this.value = boolean("disabled", false);
+          delete this.build;
+          return this;
+        }
+      },
+      {
+        name: "dismissible",
+        commit(): Attribute {
+          this.value = boolean("dismissible", false);
+          delete this.build;
+          return this;
+        }
+      },
+      {
+        name: "height-scale",
+        commit(): Attribute {
+          this.value = select("heightScale", scale.values, scale.defaultValue);
+          delete this.build;
+          return this;
+        }
+      },
+      {
+        name: "loading",
+        commit(): Attribute {
+          this.value = boolean("loading", false);
+          delete this.build;
+          return this;
+        }
+      },
+      {
+        name: "intl-close",
+        commit(): Attribute {
+          this.value = text("intlClose", TEXT.close);
```

Comment:
there is no `TEXT.close` in `flow-item.resources.ts`. Either add it to the resources file or hardcode a value this line.

Reply:
Yikes, good catch! I went with reusing panel's resources file as `flow-item` delegates to it.